### PR TITLE
feat: add about us modal

### PR DESCRIPTION
**Pull request by elysee15**
Capture d’écran 2021-09-07 à 17 28 43

### CODE REVIEW

**Comment by bbengfort** [comment]
@elysee15 I updated the modal copy:

Screen Shot 2021-09-07 at 13 28 17

File: web/src/components/AboutUs.tsx [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+				<footer>
+					<Box display="flex" flexDirection="column" alignItems="center">
+						<Box display="flex" alignItems="center" gridGap=".5rem" marginBottom=".3rem">
+							{status.status && status.version && (
+								<>
+									<Typography variant="caption">
+										{status.status && (
+											<Tooltip title={status.host || ""} style={{ cursor: "pointer" }}>
+												<span>
+													<Badge content="server status" color={StatusColor[status?.status]} />
```
Comment (bbengfort) [inline web/src/components/AboutUs.tsx:61]: Should we remove the tooltip and simply have the Badge content be the hostname? I think the tooltip was more important when it was in the footer, but in the about modal, it seems like an additional action (click about us then hover) to get the information that is unnecessary. What do you think?

File: web/src/components/AboutUs.tsx [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+						Team!) We&apos;ve made our releases and the code freely available under the{" "}
+						<a href="https://github.com/rotationalio/whisper/blob/main/LICENSE">Apache License 2.0</a> and we&apos;d feel
+						privileged if you used Whisper in your own organization. Please note, however, that Rotational Labs makes no
+						guarantees about the security of this software project and provides all code and binaries as is for general use.
+						Use with common sense and at your own risk!
+					</p>
+					<p>
+						If you&apos;re a Rotational customer and are interested in Whisper, please let us know, we&apos;d be happy to deploy
+						it for you as a single-tenant service. If you&apos;re not a Rotational customer but are interested, please get in
+						touch with us at <a href="mailto:info@rotational.io">info@rotational.io</a>.
```
Comment (bbengfort) [inline web/src/components/AboutUs.tsx:49]: The mailto link doesn't seem to be working for me; is it working for you? Is this a react thing?

**Review by bbengfort — APPROVED** [review]
Nice one!

File: web/src/components/AboutUs.tsx (hunk 2 above)
Comment (elysee15) [reply web/src/components/AboutUs.tsx:49]: No, it should work normally. On my side it has opened gmail

Comment (bbengfort) [reply web/src/components/AboutUs.tsx]: Must have just been me then.

File: web/src/components/AboutUs.tsx [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+				</div>
+				<footer>
+					<Box display="flex" flexDirection="column" alignItems="center">
+						<Box display="flex" alignItems="center" gridGap=".5rem" marginBottom=".3rem">
+							{status.status && status.version && (
+								<>
+									<Typography variant="caption">
+										{status.status && (
+											<span>
+												<Badge content="server status" color={StatusColor[status?.status]} />
```
Comment (bbengfort) [inline web/src/components/AboutUs.tsx:61]: Actually, I was thinking more along the lines of:

```
<Badge content=`${status?.host || "server"} status` color={StatusColor[status?.status]} />
```

I take it you prefer this to the tooltip?

**Review by bbengfort — APPROVED** [review]
The tests aren't passing for some reason; hoping it's a transient thing and I'm restarting the tests now.

**Comment by bbengfort** [comment]
@elysee15 would you mind taking a look at the tests? I've rerun them, but I'm not sure why they're failing.

File: web/src/components/AboutUs.tsx (hunk 2 above)
Comment (elysee15) [reply web/src/components/AboutUs.tsx]: Maybe

**Pull request closed without merging**